docs/workflow/gitlab.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "GitLab"
+toc: true
+weight: 592
+indent: true
+---
+
+# Continuous Delivery with GitLab
+
+## GitLab
+[GitLab][GitLab Kubernetes docs] supports deploying Kubernetes applications
+using GitLab pipelines today, but cloud service dependencies must be
+separately provisioned, connected, and secured. 
```

Comment:
I think we could provide a little more context here. This is succinct to the extent that it comes across to me more like a criticism of GitLab than (what I assume should be) an introduction to the value Crossplane can provide atop GitLab.

Reply:
this is the same intro GitLab is including in their 12.5 release notes:
https://gitlab.com/gitlab-com/www-gitlab-com/merge_requests/34721/diffs